data/orchestrator.yaml

```diff
@@ -3,7 +3,7 @@ orchestrator::base_dir: "%{hiera('common::base_dir')}"
 
 aem_orchestrator::jarfile_source: "s3://%{::data_bucket}/%{::stackprefix}/aem-orchestrator-%{::aem_orchestrator_version}.jar"
 aem_orchestrator::manage_installdir: true
-aem_orchestrator::application_properties::aws_sns_topic_name: "%{::stackprefix}-aem-asg-event-topic"
+aem_orchestrator::application_properties::aws_cloudformation_stack_name_messaging: "%{::stackprefix}-aem-messaging-stack"
```

Comment:
the stack name is configuration in the stack builder. not sure if we can hardcode it to be aem-messaging-stack. might have to for the time being though.